stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/TypeAdapterGenerator.java

```diff
@@ -245,11 +246,6 @@ private static MethodSpec getReadMethodSpec(@NotNull TypeName typeName,
                         "();\n" +
                         "\twhile (reader.hasNext()) {\n" +
                         "\t\tString name = reader.nextName();\n" +
-                        "\t\tcom.google.gson.stream.JsonToken jsonToken = reader.peek();\n" +
-                        "\t\tif (jsonToken == com.google.gson.stream.JsonToken.NULL) {\n" +
-                        "\t\t\treader.skipValue();\n" +
-                        "\t\t\tcontinue;\n" +
-                        "\t\t}\n" +
```

Comment:
1. True, the option of the server not sending data slipped my mind. And if you truly want to enforce a field always being filled, you can just put the `@NonNull` annotation on your field and stag will verify it. Okay, so that's not a problem.
2. Okay that's a good idea then, as we will be comparable with gson. 

The static method with a default value is also a good idea 👍